contrib/syntax-checking/packages.el

```diff
@@ -147,5 +148,12 @@
     :commands helm-flyspell-correct
     :init (evil-leader/set-key "Sc" 'helm-flyspell-correct)))
 
+(defun syntax-checking/post-init-evil ()
+  (add-to-list 'evil-motion-state-modes 'flycheck-error-list-mode)
+  (evil-define-key 'motion flycheck-error-list-mode-map
+    (kbd "RET") #'flycheck-error-list-goto-error
```

Comment:
When I tried just `"RET"`, it bound R-E-T (R followed by E followed by T) instead of binding the return key. Wrapping in `kbd` was the solution that worked.
Do you know that <kbd>C-g</kbd> closes popup windows such as flycheck's error list? It doesn't bury the buffer, so it may cause issues with <kbd>SPC TAB</kbd> or others. Also, may I suggest making <kbd>el</kbd> a toggle, instead of opening with <kbd>el</kbd> and closing with <kbd>eL</kbd>? It may be more comfortable.

P.S. "bury" is a technical term